core/src/main/res/values/arrays.xml

```diff
@@ -13,16 +13,22 @@
         <item>never</item>
     </string-array>
 
-    <string-array name="spnVolumeReductionItems">
+    <string-array name="spnVolumeAdaptationItems">
         <item>@string/feed_volume_reduction_off</item>
         <item>@string/feed_volume_reduction_light</item>
         <item>@string/feed_volume_reduction_heavy</item>
+        <item>@string/feed_volume_boost_light</item>
```

Comment:
Logically, I think "off" fits better between reduction and boost. Would moving it here break anything? I think it should work